es6-destructuring/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ES6 Destructuring</title>
+</head>
+<body>
+  <script>
+    const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+    let title = book1.title;
+    let author = book1.author;
+    let libraryID = book1.libraryID;
+
+    console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+
+    const book2 = {
+      title: 'Les Fleurs du mal',
+      author: 'Charles Baudelaire',
+      libraryID: 2345
+};
+    title = book2.title;
+    author = book2.author;
+    libraryID = book2.libraryID;
+
+    console.log(`The title of the book is ${title}, the author is ${author}, and the library id is ${libraryID}`);
+
+    const library = [
+      {
+        title: 'The Road Ahead',
+        author: 'Bill Gates',
+        libraryID: 1254
+      },
+      {
+        title: 'Walter Isaacson',
+        author: 'Steve Jobs',
+        libraryID: 4264
+      },
+      {
+        title: 'Mockingjay: The Final Book of The Hunger Games',
+        author: 'Suzanne Collins',
+        libraryID: 3245
+      },
+      {
+        title: "Dead Dead Demon's De De De De Destruction",
+        author: 'Inio Asano',
+        libraryID: 1233
+      }
+    ];
+
+    const book3 = library[0];
+    const book4 = library[1];
+    const book5 = library[2];
+    console.log('book3: ', book3);
+    console.log('book4: ', book4);
+    console.log('book5: ', book5);
+    const book6 = library[3];
+    console.log('book6: ', book6);
```

Comment:
Please use destructuring to get your values. Re-read the instructions and external documentation if you don't know what that is.

Reply:
![image](https://user-images.githubusercontent.com/97073517/174711624-8effd49f-b7ef-443c-84f2-e9ea04e92908.png)